go/sqlescape/ids_test.go

```diff
@@ -26,12 +29,182 @@ func TestEscapeID(t *testing.T) {
 	}, {
 		in:  "a`a",
 		out: "`a``a`",
+	}, {
+		in:  "`fo`o`",
+		out: "```fo``o```",
+	}, {
+		in:  "",
+		out: "``",
 	}}
 	for _, tc := range testcases {
-		out := EscapeID(tc.in)
-		if out != tc.out {
-			t.Errorf("EscapeID(%s): %s, want %s", tc.in, out, tc.out)
-		}
+		t.Run(tc.in, func(t *testing.T) {
+			out := EscapeID(tc.in)
+			assert.Equal(t, out, tc.out)
+		})
+	}
+}
+
+func TestUnescapeID(t *testing.T) {
+	testcases := []struct {
+		in, out string
+		err     bool
+	}{
+		{
+			in:  "``",
+			out: "",
+			err: false,
```

Comment:
I think this case should also error since the empty string is not a valid identifier.

Reply:
done